bbc/simorgh
# Updating package lock; Adding core-js as devDependency

No related Issue
**Background**
Upon removing `node_modules` &  doing an `npm install` to update the `package-lock.json`.

**This PR**

- Updating the `.nvmrc` file to include the patch version that's used on infrastructure
- Updating the package lock
After clearing the local cache `npm cache clean --force`, there were missing core-js dependencies for tests.

- Added `core-js` as a devDependency. 
- Removed `--only=production` from the `npm ci` command since it isn't supported by npm. 

You should run these commands and get these versions: 
`node -v` `v8.11.3`
`npm -v` `6.4.1`

If you have different versions:

```
nvm install 8.11.3
nvm use 8.11.3
npm i -g npm@6.4.1
npm cache clean --force
```
within simorgh:

```
rm -rf node_modules package-lock.json && npm install
```


**Testing instructions**
With a clean install, all tests should pass.

```
cd simorgh && git checkout update-packagelock && rm -rf node_modules && npm ci && npm run test
```
